canonical-web-and-design/snapcraft.io
# Add remove repo

## Done

- Add disconnect repo

## Issue / Card

Fixes #2650 

## QA

- Pull the branch
- Run the site using the command `./run`
- View the site locally in your web browser at: http://0.0.0.0:8004/<snap_name>/builds
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Go to a snap with builds and click on `Disconnect repo` link and then click `Confirm` button
- See the repo has been disconnected
- Perhaps this should be QAed from different GitHub accounts as well.
